packages/docsearch-react/src/DocSearchModal.tsx

```diff
@@ -328,6 +335,18 @@ export function DocSearchModal({
     }
   }, [initialQuery, refresh]);
 
+  React.useEffect(() => {
+    // Inside of a "useEffect" hook add an event listener that updates
```

Comment:
We can remove this comment, it's pretty explicit since `addEventListener` and `resize` are native browser APIs.